### PR TITLE
CT-330 Add Whose it With

### PR DESCRIPTION
**Pull request by aliuk2012**
https://dsdmoj.atlassian.net/browse/CT-330

At first it was decided that we should be showing business units
but as we do not have business units implemented we have decided
to use the drafters email address as a clue to who the case is
currently owned by. If it is not assigned to a KILO then it
displays 'DACU' as the Business Management Team (BMT) dealing
with the case.

Later this implementation will be swapped out for Business Units.

### CODE REVIEW

File: app/models/case.rb [hunk 1]
```diff
@@ -150,6 +150,14 @@ def attachments_dir(attachment_type)
     "#{Digest::SHA1.hexdigest(id.to_s)}/#{attachment_type}"
   end
 
+  def whose_it_with
```
Comment (EdwardAndress) [inline app/models/case.rb]: Should this be `who_its_with`?

Comment (aliuk2012) [inline app/models/case.rb]: 👍

File: app/views/cases/index.html.slim [hunk 2]
```diff
@@ -26,6 +27,8 @@
           = t('.external_deadline')
         th scope='col'
           = t('.status')
+        th scope='col'
+          = "Whose it with"
```
Comment (EdwardAndress) [inline app/views/cases/index.html.slim]: Does this need to be updated too?

Comment (aliuk2012) [inline app/views/cases/index.html.slim]: (Doh) Thats should have been in locales. Well spotted

**Pull request merged**